OJP/OJP_JourneySupport.xsd

```diff
@@ -410,6 +410,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="StopCallStatusGroup"/>
+			<xs:element name="Attribute" type="GeneralAttributeStructure" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Note or attributes.</xs:documentation>
```

Comment:
Why note singular and attributes plural?

Reply:
changed